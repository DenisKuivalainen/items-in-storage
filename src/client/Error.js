import React from 'react';
import { Typography } from '@material-ui/core';

const Error = (message) => (
    <div style={{paddingTop: 100, width: '100%', textAlign: 'center'}}>
        <Typography variant="h4" color="error">
            {message}
        </Typography>
    </div>
);

export default Error;