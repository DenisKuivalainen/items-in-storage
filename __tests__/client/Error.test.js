import React from 'react';
import renderer from 'react-test-renderer';
import Error from '../../src/client/Error';

describe("test Error ", () => {
    test("sanpshot with message 'This is error'", () => {
        const tree = renderer.create(Error("This is error")).toJSON();
        expect(tree).toMatchSnapshot();
    });
})