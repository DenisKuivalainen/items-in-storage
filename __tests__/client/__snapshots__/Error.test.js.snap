// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Error  sanpshot with message 'This is error' 1`] = `
<div
  style={
    Object {
      "paddingTop": 100,
      "textAlign": "center",
      "width": "100%",
    }
  }
>
  <h4
    className="MuiTypography-root MuiTypography-h4 MuiTypography-colorError"
  >
    This is error
  </h4>
</div>
`;
